tests/integration/test_vulnerability_detector/test_feeds/test_invalid_syntax_redhat_feeds.py

```diff
@@ -93,11 +93,14 @@ def modify_feed(test_data, request):
     """
     Modify the redhat OVAL feeds, setting a test field value
     """
-    backup_data = read_json_file(custom_redhat_oval_feed_path)[0]
+    backup_data = read_json_file(custom_redhat_oval_feed_path)
 
-    modified_data = dict(backup_data)
+    modified_data = json.dumps(dict(backup_data[0]))
 
-    modified_string_data = replace_regex_group(test_data['pattern'], test_data['update'], str(modified_data))
+    modified_string_data = replace_regex_group(pattern=test_data['pattern'], new_value=test_data['update'],
+                                               data=modified_data)
+
+    modified_string_data = '[\n' + modified_string_data + ',\n' + json.dumps(backup_data[1]) + '\n]'
```

Comment:
Same here, it would be nice to change this to a fstring.

Reply:
Done in f3a1c5d2899020f0620c96c8a9cf2d596245f804